OrbitGl/CaptureDeserializer.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_CAPTURE_DESERIALIZER_H_
+#define ORBIT_GL_CAPTURE_DESERIALIZER_H_
+
+#include <google/protobuf/io/coded_stream.h>
+#include <google/protobuf/io/zero_copy_stream_impl.h>
+#include <google/protobuf/message.h>
+
+#include <iosfwd>
+#include <outcome.hpp>
+#include <string>
+
+#include "CaptureData.h"
+#include "OrbitBase/Result.h"
+#include "TimeGraph.h"
+#include "capture_data.pb.h"
+
+class CaptureDeserializer {
```

Comment:
As a follow-up (in a different PR) and a simple first step into also making the Deserializer independent, can you also make this class to be just a namespace with static methods and pass the time_graph_ as parameter?

Reply:
Sure, makes sense; will do that next in a separate PR, thanks!